tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -38,6 +38,7 @@
 
 # pylint: disable=no-name-in-module,import-error
 from openvino.frontend import FrontEndManager, OpConversionFailure, TelemetryExtension
+from openvino.frontend.pytorch.module_extension import ModuleExtension
```

Comment:
@mvafin, what does happen if `-DENABLE_OV_PYTORCH_FRONTEND=OFF` provided in cmake configuration? This is a direct dependency to a specific FE. Do we have other examples of such imports somewhere else?

Reply:
Since this is a python code, I would expect it to be available even when `-DENABLE_OV_PYTORCH_FRONTEND=OFF`

Reply:
Below there is another direct import of `ModuleExtension`. And in the same file we have all other import of PT FE specific modules only inside functions under try-except, so `ModuleExtension` should be imported in the same way.

Reply:
`ModuleExtension` does not have any dependencies, so importing it does not break anything